iron-iconset-svg.html

```diff
@@ -225,8 +230,9 @@
           cssText += '-webkit-transform:scale(-1,1);transform:scale(-1,1);';
         }
 
-        svg.setAttribute('viewBox', viewBox);
-        svg.setAttribute('preserveAspectRatio', 'xMidYMid meet');
+        svg.setAttributeNS(null, 'viewBox', viewBox);
```

Comment:
Not needed at all: it was a workaround for https://github.com/webcomponents/custom-elements/issues/14 (fixed) that I totally forgot about.